modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
+.. In the canvas, click *Create Notification*.
+.. In the *Create Notification* properties window, click the green plus icon (*+*) to the right of the *Connector configuration* field.
+.. In the *Global Element Properties* window, configure authentication to Anypoint Platform and click *OK*:
++
+.The arrow shows the *Username* field in the *Global Element Properties* window.
+image::alerts-custom-ch-connector.png[Username field in the Global Element Properties window]
++
+The connector can use basic authentication or inherited token authentication.
+.. In the *Create Notification* properties window, enter the CloudHub app name in the *Domain* field and the notification in the *Message* field:
++
+.The arrow shows the *Message* field in the *Create Notification* properties window.
+image::alerts-custom-ch-connector-notification.png[Message field in the Create Notification properties window]
++
+For more information, see 
+xref:connectors::cloudhub-connector/1.0/#creating-notifications[Creating a Notification].
+--
+. Deploy the app to CloudHub.
+. In Runtime Manager, bind the custom notification to an alert:
++
+--
+.. Click *Alerts* in the menu on the left. 
+.. Click the blue plus icon (*+*) to create a new alert.
+.. In the *Create an Alert* page, configure your custom alert and click *Submit*:
++
+.The arrow shows the *Applications* field in the *Create an Alert* window.
+image::alerts-custom-rtm-create.png[Applications field in the Create an Alert window]
++
+Name::
+Enter a meaningful name for the custom alert. 
+Severity level::
+Select the severity level to apply to the alert:
+* *Critical*
+* *Warning*
+* *Info*
+Source::
+Select *Applications* for the alert source.
+Application type::
+Select *CloudHub Applications*.
+Applications::
+Select the application that contains the flow that you configured to trigger a custom alert.
+Condition::
+Select *Custom application notification*.
+** Select the priority from the *Priority* menu, or leave the priority as *Any* to send notifications regardless of priority.
+** In *Contains*, enter the string to use to trigger a notification.
++
+This string must match a string in the *Message* field that you configured in CloudHub Connector.
+Subject::
+Enter the subject line to include in the notification email. 
+Message::
+Enter the custom message to include in the notification.
+Recipients::
+Enter the email addresses for notification recipients. 
 
-Once your application is running, you can trigger the notification by going to your application's URL, for example: `+http://YOURAPP.cloudhub.io/hello-notification+`. The page displays a `Hello World` message, and automatically creates a new notification in your Runtime Manager console. 
++
+You can use variables in the alert *Subject* and *Message* fields.
+For information, see xref:alerts-on-runtime-manager.adoc#alert-variables[Alert Variables].
```

Comment:
ditto re. specificity

Reply:
Ditto redundant, but OK

`For information about using variables in alerts, see xref:alerts-on-runtime-manager.adoc#alert-variables[Alert Variables].
`